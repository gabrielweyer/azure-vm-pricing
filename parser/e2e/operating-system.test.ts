import assert from "./assert-extensions";

describe('End-to-end tests for supported operating systems', () => {
  const culture = 'en-us';
  const currency = 'usd';

  test('Ubuntu Advantage Essential', (done) => {
    const operatingSystem = 'ubuntu-advantage-essential';

    assert(done, culture, currency, operatingSystem);
  });
});
