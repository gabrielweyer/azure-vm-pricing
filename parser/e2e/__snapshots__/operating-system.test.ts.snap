// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`End-to-end tests for supported operating systems Ubuntu Advantage Essential 1`] = `"D2 v3,2,8,0.126,N/A,0.0883,N/A,0.0598,N/A,0.0209,N/A"`;
